components/raftstore/src/store/fsm/apply.rs

```diff
@@ -942,6 +965,10 @@ where
                     }
                 }
             }
+            if has_high_latency_operation(&cmd) && apply_ctx.priority != Priority::Low {
+                self.priority = Priority::Low;
+                return ApplyResult::Yield;
+            }
```

Comment:
How about change it priority to `Normal` ?
```
if has_high_latency_operation() {
    self.priority = Priority::Low;
    if apply_ctx.priority != Priority::Low {
        return ApplyResult::Yield;
    }
}  else {
   self.priority = Priority::Normal;
}
```

Reply:
There is a little difference. Changing priority to normal here may not take effect immediately unless yield also, because current fsm may have already scheduled to the low-priority PollHandler due to previous high latency operations. 

We can simply change priority to normal after current fsm has processed finished, so that new batch msgs can always be scheduled to normal-priority PollHandler first.

Reply:
OK. But you must commit the current delete if there is still some data kept in WriteBatch, which is written by previous entries.

Reply:
Yes, they have been committed before this lines.